src/cli.js

```diff
@@ -17,6 +23,10 @@ changelog will be ignored.
 If the '--rc' flag is used and the section for the current release does not \
 yet exist, it will be created.`;
 
+const validateEpilog = `This does not ensure that the changelog is complete, \
+or that each change is in the correct section. It just ensures that the \
+formatting is correct. Verification of the contents is left for manual review.`;
```

Comment:
Aside: I've been wondering with increasing frequency if backticks / template strings are superior to single/double-quoted strings and concatenation using `+`. I'm starting to think that may not be the case 🤔

Reply:
Oh? I find them especially useful with longer strings. I like not having to use `\n` anywhere.

Reply:
String style fight! String style fight! String style fight!